### PR TITLE
Update Hints and Tips: How to update plugins to Trados Studio 2022

### PR DESCRIPTION
**Pull request by Impertatore**

### CODE REVIEW

File: articles/hints_tips/Update_Plugins/update_plugins_to_Studio2022.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+<br/>
+
+### Migrate packages.config to PackageReference
+If you project is still using a packages.config file to reference the nuget packages, then we need to migrate it to PackageReference from the visual studio solution.  Right-click on the **packages.config** file from the project solution and select **Migrate packages.config to PackageReference...** from the context menu.
+<img style="display:block; margin: 0px" src="images/MigratePackagesConfig.png" />
+
+<br/>
+
+### Remove redundant references to the plugin framework
+Unload the project from visual studio and remove the obsolete references to the plugin framework as highlighted underneath:
```
Comment (ituriSdl) [inline articles/hints_tips/Update_Plugins/update_plugins_to_Studio2022.md:123]: Please use uppercase for Visual Studio

**Review by ituriSdl — COMMENTED** [review]
I think it would be nice if we add a link to this article directly from the landing page, that is https://github.com/RWS/studio-api-docs/blob/main/index.md/#L1

File: articles/hints_tips/Update_Plugins/update_plugins_to_Studio2022.md (hunk 1 above)
Comment (Impertatore) [reply articles/hints_tips/Update_Plugins/update_plugins_to_Studio2022.md:123]: used uppercase for Visual Studio

**Review by paulfilkin — APPROVED** [review]
Agree that a link from the main page is a good idea. Otherwise looks good to me.

**Pull request merged**